lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2015 SUSE Linux GmbH
+# Copyright (C) 2016 SUSE LLC
```

Comment:
@aaannz where is your copyright checker script? Don't waste manual review time with this